packages/truffle/test/scenarios/sandbox.js

```diff
@@ -15,7 +15,10 @@ module.exports = {
       path.join(tempDir.name, subPath, "truffle-config.js"),
       {}
     );
-    return config;
+    return {
+      config,
+      cleanupCallback: tempDir.removeCallback
```

Comment:
`cleanupSandboxDir` would communicate more context on the client side.

Reply:
sounds good